### DISCUSSION TITLE
Disabling star bursts in default skins

### DISCUSSION THREAD

**Discussion by aalex03** [ppy/osu#29792] [Q&A] [answered]
So the new update added star fountains/bursts to the game.
I know they can be removed from normal skins by making the respective file a blank image but I haven't seen any option of turning them off for the default lazer skins. Is it possible?
I'm guessing adding more and more toggles for every feature like this is not the way the devs want to handle these and it will probably be addressed with updates to the skin editor, but hey as a compromise I suggest removing them from the argon pro skin.

**Comment by Dimmitsaras** [comment]
Huge miss here, argon is split into two skins and pro is supposed to get rid of distractions. What's the point in hiding "Great" when we get stars blasting on our screen instead?

So default skins can just not disable them. And even for non-default skins why is the user expectation to go out their way to delete files? Stable does this with a simple toggle..

**Comment by felipemarins** [comment]
I agree that argon pro should not include the star bursts. Also, I think that the default argon skin could maybe include an updated design for the star bursts, as the current one is the same used on the classic skin and I don't think it matches well with the argon design. Maybe instead of a star burst, the screen corners could flash, but I don't know exactly if it would be good.

**Reply by Tenexxt** [reply]
> Maybe instead of a star burst, the screen corners could flash, but I don't know exactly if it would be good.

Since the falshes would have to be on Kiai, and on beat - This sounds like a perfect thing to give someone with epiliepsy health problems on high BPM maps because of a in-game cosmetic (that of course depends on its intensivity, but i feel like flashing someone in a rhythm game is not a good idea anyway).

**Reply by felipemarins** [reply]
It wouldn't be an intense flash and it would be played only on the start of kiai time like the current star bursts (as it is meant to be a replacement for star bursts on argon).

**Comment by shiumano** [comment]
The default skin doesn't have star particles in Kiai or during breaks, so it seems different to have starbursts.

**Comment by peppy** [comment, marked as answer]
I made a [toggle for this](https://github.com/ppy/osu/compare/master...peppy:osu:star-fountain-toggle?expand=1) but I'm not sure this is the correct direction. As proposed, I think it may be best to limit this to classic skins for now, and make some new more modern visualisations for the newer skins.

**Reply by peppy** [reply]
Related: https://github.com/ppy/osu/discussions/29830

**Reply by SupDos** [reply]
As someone using a classic skin, I love having the fountains in the main menu but don't want to see them in gameplay. Blanking out the `star2.png` file would remove them everywhere which is not what I'd want.

To avoid having an extra toggle, maybe it could be combined with the combo burst image toggle (whenever that is implemented) as a "hide burst effects during gameplay" option, hiding both combo bursts and kiai fountains. 2 birds with 1 stone kinda?

**Reply by felipemarins** [reply]
I agree with the idea of only hiding it during gameplay, as the reasoning for removing it is only to reduce distractions while playing.
